Server/VikingVault.API/Controllers/LoginController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
+using VikingVault.Services.Abstractions;
+using VikingVault.DataAccess.Models;
+
+namespace WebApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")]
+    public class LoginController : ControllerBase
+    {
+        private ILoginService _loginService;
+
+        public LoginController(ILoginService loginService)
+        {
+            _loginService = loginService;
+        }
+
+        [AllowAnonymous]
+        [HttpPost("authenticate")]
```

Comment:
Adding an explicit route name is kind of redundant here, as the name is implied from the method name "Authenticate" in here, also I see that you're not explicitly custom route names in other controllers, so it's better to keep it as consistent as possible (either you add them in all the places or let the naming be implied from the controller methods)